frontend/src/Leaderboard/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { ScrollView } from 'react-native';
+import RankRow from './RankRow';
+
+const mockedUsers = [
+  {
+    username: 'userA',
+    gold: 10,
+    kingdom: 10,
+  },
+  {
+    username: 'userB',
+    gold: 12,
+    kingdom: 12,
+  },
+  {
+    username: 'userCCCCCCCCCCCcCCC',
+    gold: 12,
+    kingdom: 12,
+  },
+];
+
+function Leaderboard() {
+  return (
+    <ScrollView>
+      {
+        mockedUsers.map((user, idx) => (
+          <RankRow
+            key={user.username}
+            rank={idx}
```

Comment:
```suggestion
            rank={idx + 1}
```

right now ranking starts at 0, but it should start with 1. 
Adding +1 will solve it, please accept it.
You can accept it here on GitHub by clicking on the `Commit suggestion` button, but pay attention to the commit message here too, use the same format as always!

![Screen Shot 2019-12-11 at 16 38 45](https://user-images.githubusercontent.com/30794349/70605143-db385280-1c34-11ea-9436-7ad7ea6c1586.png)